docs: update images using latest UI screenshots (#1992)
Unleash/unleash: website/docs/user_guide/environments.md

### Step 1: Enable new environments for your Project

> Navigate to the project and choose the “Environments” tab.

Reviewer: Unsure about this change too

Author: ```suggestion
Navigate to the project and choose the “environments” tab.
```

> Navigate to the project and choose the “Environments” tab.

Reviewer: ```suggestion
Navigate to the project and choose the “environments” tab.
```

![A feature toggle strategies tab showing three different environments, of which one is active. The UI displays data about the currently selected environment, ](/img/environments_strategies.png 'Add strategy configuration per environment')

### Step 3: Create environment specific API keys

> @Thomas - please double-check if the image si ok with the  text bellow

Reviewer: Sure, that looks about right, but the alt text will need to be updated.

```suggestion
```

In order for the SDK to download the feature toggle configuration for the correct environment you will need to create an API token for a defined environment.

> ![An API key creation form. The form's fields are "username", "token type", "project", and, crucially, "environment". The development environment is selected.](/img/environments_api_keys.png 'Create Environment specific API Keys')

Reviewer: ```suggestion
![An API token creation form and how to get there in the Admin UI. The steps are: navigate to Configure > API access > New API token.](/img/environments_api_keys.png 'Create Environment specific API Keys')
```
